testing/regress/environment.xml.in

```diff
@@ -20,6 +20,20 @@
         storageWrites="0.0038"/>
  </Hardware>
  <Software>
+  <Globals noCommon="true"
+           maxConnections="-1">
+           <StoragePlane name="storagePlane"
```

Comment:
Doesn't contain StoragePlaneList

Also, I suspect these should not be checked in as part of the PR.

Reply:
Removed initfiles changes from PR.